app/src/main/res/layout/activity_about__us.xml

```diff
@@ -3,8 +3,8 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:background="@drawable/hill_background"
     android:layout_height="match_parent"
+    android:background="@drawable/hill_background"
```

Comment:
Is this change really required? I am no expert in VCS but won't these cause mere Merge conflicts.

Reply:
I know you are not an expert in VCS. No, they won't cause merge conflicts.

Reply:
Let's not be too sure about that buddy. ;) You will understand.